kotlin/feature-layer-query/README.md

```diff
@@ -2,15 +2,15 @@
 
 Find features in a feature table which match an SQL query.
 
-![](feature-layer-query.png)
+![Image of feature layer query](feature-layer-query.png)
 
 ## Use case
 
 Query expressions can be used in ArcGIS to select a subset of features from a feature table. This is most useful in large or complicated data sets. A possible use case might be on a feature table marking the location of street furniture through a city. A user may wish to query by a TYPE column to return "benches". In this sample, we query a U.S. state by STATE_NAME from a feature table containing all U.S. states.
 
 ## How to use the sample
 
-Input the name of a U.S. state into the text field. When you click "search", a query is performed and the matching features are highlighted or an error is returned.
+Input the name of a U.S. state into the text field. When you tap "GO", a query is performed and the matching features are highlighted or an error is returned.
 
 ## How it works
 
```

Comment:
For step 2 for Kotlin, we should replace `setWhereClause` with `whereClause` so that we are following Kotlin syntax